src/assets/scss/elements/ys-flow-steps.scss

```diff
@@ -4,22 +4,32 @@
 @import '../tools/ys-functions';
 
 .ys-flow-steps {
+  $variant-vertical-ref: 0;
+  $bullet-size-px: 34;

```

Comment:
Should use the increment of 8 as per the grid spacing/spec
https://dna.yousee.dk/docs/design-guidelines/grid.html

> Uses increments of 8 to size and space out the elements on a page. This means that any defined height or width, margin or padding will be an increment of 8 (pixels).

Reply:
I agree with you in the other cases, but in this case I would be changing the current size - which I don't want to touch.